### DISCUSSION TITLE
How to set the "parse_parallelism" flag in LcovMerger

### DISCUSSION THREAD

**Discussion by m-delattre** [bazelbuild/bazel#22974] [Q&A] [answered]
I would like to set the "parse_parallelism" flag I found in the LcovMerger code here:
https://github.com/bazelbuild/bazel/blob/master/tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/LcovMergerFlags.java#L57)
How to set this flag from the bazel command? I tried passing in to bazel coverage but it gets rejected.

**Comment by fmeum** [comment]
@c-mita

**Comment by c-mita** [comment, marked as answer]
I don't think you can; I  believe it was added largely for testing for internal purposes and I don't think anything was wired up so you could set this flag from Bazel, at least for generation of the combined coverage report.
